### PR TITLE
TDI-38346:Refactor WebServiceInsert.jar form Ant to Maven.

### PR DESCRIPTION
**Pull request by pyzhou-talend**
https://jira.talendforge.org/browse/TDI-38346

### CODE REVIEW

**Comment by pyzhou-talend** [comment]
I've tried to build the maven project on my own PC. And it works.

File: main/plugins/org.talend.designer.components.libs/libs_src/talend-webservice-input/pom.xml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd http://java.sun.com/products/javabeans/glasgow/jaf.html">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.talend.libraries</groupId>
+  <artifactId>talend-webservice-input</artifactId>
+  <version>1.0.0-20170301</version>
+  <packaging>jar</packaging>
+
+  <name>talend-webservice-input</name>
+  <url>http://maven.apache.org</url>
```
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.libs/libs_src/talend-webservice-input/pom.xml]: what does this `url` means ?

File: main/plugins/org.talend.designer.components.libs/libs_src/talend-webservice-input/pom.xml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+				<directory>src/main/java</directory>
+			</resource>
+		</resources>
+		<plugins>
+			<plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>2.3.2</version>
+                <configuration>
+                	<source>${java.source.version}</source>
```
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.libs/libs_src/talend-webservice-input/pom.xml]: I didn't find the declaration about  `java.source.version`

**Comment by pyzhou-talend** [comment]
I have remove the url. And I checked the previous jar was compiled with java1.5. So I set java.source.version into 1.5

**Comment by pyzhou-talend** [comment]
And I've confirmed the name with Pierre.

**Pull request merged**